recharges/models.py

```diff
@@ -4,14 +4,16 @@
 from django.db import models
 
 
-class DummyModel(models.Model):
+class RechargeModel(models.Model):
 
     """
     - Nice docstring please
```

Comment:
You should add a description of what the Model is for here. Something like:
`Stores all recharge request data.`  We'll probably update this with more info as we go.